spec/models/embedded_ansible_worker/runner_spec.rb

```diff
@@ -10,7 +10,9 @@
     let(:runner) {
       worker
       allow_any_instance_of(described_class).to receive(:worker_initialization)
-      described_class.new(:guid => worker_guid)
+      r = described_class.new(:guid => worker_guid)
+      allow(r).to receive(:worker).and_return(worker)
```

Comment:
This is only necessary because of the stub above.  Instead can we just tap the `.new` and call `#find_worker_record`?

`#initialize` calls: https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_worker/runner.rb#L51
`#worker_initialization` calls: https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_worker/runner.rb#L58
`#starting_worker_record` calls: https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_worker/runner.rb#L170
`#find_worker_record` sets `@worker`: https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_worker/runner.rb#L165